File: src/main/java/com/todo/javatodo/repository/PriorityRepository.java
package com.todo.javatodo.repository;

import com.todo.javatodo.entity.Priority;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;


@Repository
public interface PriorityRepository extends JpaRepository<Priority, Long> {


    List<Priority> findByUserEmailOrderByIdAsc(String email);


}

File: src/main/java/com/todo/javatodo/repository/PriorityRepository.kt
package com.todo.javatodo.repository

import com.todo.javatodo.entity.Priority
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PriorityRepository : JpaRepository<Priority, Long> {
    fun findByUserEmailOrderByIdAsc(email: String): List<Priority>
}